### PR TITLE
New project introduction

### PR DESCRIPTION
**Pull request by rachelzhang0922**
#### Which issue(s) this PR fixes:

Fixes #

#### How is this PR tested:
- [ ] unit test
- [ ] e2e test
- [ ] other (please specify)

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: docs/community/contribute-shifu.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
 ---
 title: Contribute to Shifu
-sidebar_position: 1
+sidebar_position: 5
 ---
 
-# Contribute to Shifu
+## Contribute to Shifu
```
Comment (tomqin93) [inline docs/community/contribute-shifu.md:6]: shouldn't this be title with one `#` ?

File: docs/introduction.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+---
+
+# Platform Introduction
+
+***Shifu*** is derived from the Chinese word "师父". **Teachers, therefore preach, teach and dispel doubts.**. As a next-generation open source IoT development framework, ***Shifu*** has gain control over IoT devices and successfully endow such capabilities to developers in the form of APIs. In this way, traditional IoT application development is reduced to simple web development, which greatly improves the efficiency, quality and reusability of IoT application development.
+
+## Innovation Advantages
+
+### 🌨️ Cloud-native
```
Comment (tomqin93) [inline docs/introduction.md:13]: let's use ☁️ instead of a rainy cloud

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/introduction.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+# 平台简介
+
+
+
+***Shifu***一名来源于中文“师父”。**师者，所以传道受业解惑也**。作为下一代开源物联网开发框架，***Shifu*** 实现了对物联网设备的高度控制，并成功以API的形式将这样的能力传授给开发者。这样一来，传统的物联网应用开发就被简化为简单的网络开发，从而大大提高了物联网应用开发的效率、质量和复用性。
+
+## 创新优势
+
+### 🌨️云原生
```
Comment (tomqin93) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/introduction.md:16]: same as above

File: docusaurus.config.js [hunk 4]
```diff
@@ -151,51 +151,6 @@ const config = {
...
-                label: 'GitHub',
-                href: 'https://github.com/edgenesis/shifu',
-              },
-              {
-                label: 'Download Demo',
-                href: '/disclaimer',
-              },
-            ],
-          },
-        ],
```
Comment (tomqin93) [inline docusaurus.config.js:198]: are we removing all footers?

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@rachelzhang0922 added some comment, please take a look

**Comment by tomqin93** [comment]
@rachelzhang0922 also please sync with main to address conflicts

File: docusaurus.config.js (hunk 4 above)
Comment (rachelzhang0922) [reply docusaurus.config.js:198]: yes，I've already synced with yongli

File: docs/community/contribute-shifu.md (hunk 1 above)
Comment (rachelzhang0922) [reply docs/community/contribute-shifu.md:6]: 三个标题是平级关系，如果都用一级标题貌似不太好，所以就都用二级标题了

File: docs/introduction.md (hunk 2 above)
Comment (rachelzhang0922) [reply docs/introduction.md:13]: lol

File: docusaurus.config.js (hunk 4 above)
Comment (rachelzhang0922) [reply docusaurus.config.js:199]: in the future we will move all the links to the navbar

**Review by tomqin93 — APPROVED** [review]
LGTM

**Pull request merged**